filters/color-overlay/src/ColorOverlayFilter.ts

```diff
@@ -64,6 +68,19 @@ class ColorOverlayFilter extends Filter
     {
         return this._color;
     }
+
+    /**
+     * The alpha value of the color
+     * @member {number}
+     * @default 0
+     */
+    set alpha(value: number) {
```

Comment:
Run `npm run lint -- --fix` to resolve some of these issues.

Reply:
ok, I ran it